### PR TITLE
ArgumentDefaultsHelpFormatter: fix docstring

### PR DESCRIPTION
**Pull request by ferdnyc**
In #27808 <s>@michiboo</s> @abadger enhanced the display of default values in `argparse` help messages. Unfortunately, the docstring for `argparse.ArgumentDefaultsHelpFormatter` got somewhat garbled along the way.

* Originally, commit 336bd2faf6de65055e900d027ab696f2a052d357 added a new function, `_add_defaults_to_help_string`, documented as:
    > Add the default value to the option help message.
    >
    > ArgumentDefaultsHelpFormatter and BooleanOptionalAction both want to add the default value to the help message when it isn't already present.  This code will do that, detecting cornercases to prevent duplicates or cases where it wouldn't make sense to the end user.

* But subsequent commit dc4babf8077a6cae5ecb2f8ecd20abde0c4e318b incorporated that functionality into `argparse.ArgumentDefaultsHelpFormatter._get_help_string` instead, and its docstring was set to:
    > Add the default value to the option help message.
    >
    > ArgumentDefaultsHelpFormatter and BooleanOptionalAction when it isn't already present. This code will do that, detecting cornercases to prevent duplicates or cases where it wouldn't make sense to the end user.

This PR ~clears up and simplifies~ removes that docstring, as it didn't really add information not discernible just by reading the code.

### CODE REVIEW

**Comment by ferdnyc** [comment]
@michiboo Do you have a moment to look at this? It's purely a docstring adjustment, related to your previous changes to `argparse`'s `ArgumentDefaultsHelpFormatter`.

**Comment by ferdnyc** [comment]
My apologies, the PR in question was @abadger's, based on an earlier PR by @michiboo.

@abadger, do you have a moment to look at this? It's purely a docstring adjustment, related to your previous changes to `argparse`'s `ArgumentDefaultsHelpFormatter`.

File: Lib/argparse.py [hunk 1]
```diff
@@ -703,13 +703,10 @@ class ArgumentDefaultsHelpFormatter(HelpFormatter):
     """
 
     def _get_help_string(self, action):
-        """
-        Add the default value to the option help message.
+        """Add the default value to the option help message.
```
Comment (serhiy-storchaka) [inline Lib/argparse.py]: I would convert the docstring into comment or remove it. No other private method in this class hierarhy has a docstring. And the text just repeats what the code do, but less accurately.

Comment (ferdnyc) [reply Lib/argparse.py]: @serhiy-storchaka That seems like a good idea. I was focused on the _coherence_ of the text (or lack thereof), but looking at it again I agree it doesn't really add much even when it makes sense.

Comment (ferdnyc) [reply Lib/argparse.py]: I decided to just yank it entirely, fully in agreement with:

> the text just repeats what the code do, but less accurately.

**Review by serhiy-storchaka — APPROVED** [review]

**Comment by serhiy-storchaka** [comment]
Thank you for reporting and fixing this issue.

**Pull request merged**